pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -17,59 +17,56 @@ social:
 -# Not ideal but we can pull the undigested file from /blockly.
 %script{src: CDO.studio_url('/blockly/js/tutorialExplorer.js')}
 
-- tutorialsContents = Tutorials.new(:tutorials).contents(request.host_with_port)
+- tutorials_contents = Tutorials.new(:tutorials).contents(request.host_with_port)
 
 - filters = []
 
 - entries = []
-- entries << {name: "pre", text: "Pre-reader"}
-- entries << {name: "2-5", text: "2-5"}
-- entries << {name: "6-8", text: "6-8"}
-- entries << {name: "9+", text: "9+"}
+- entries << {name: "pre", text: "Pre-reader (Ages 4-6)"}
+- entries << {name: "2-5", text: "Grades 2-5 (Ages 7-10)"}
+- entries << {name: "6-8", text: "Grades 6-8 (Ages 11-13)"}
+- entries << {name: "9+", text: "Grades 9+ (Ages 14+)"}
```

Comment:
Yes, in a later step we'll loc everything.  (For all strings inside React code, I was planning on passing in a little string table since we do loc via Ruby & gsheet in Pegasus.)